File: 2016 czerwiec/c++/3.cpp
#include <iostream>
#include <fstream>
#include <string>
using namespace std;


int main()
{
    ifstream in("liczby.txt");
    ofstream out("wyniki_6_3.txt");

    int ans = 0;
    for (int i = 1; i <= 999; i++)
    {
        string s;
        in >> s;
        if (s[s.size() - 1] == '2' && s[s.size() - 2] == '0')
            ans++;
    }
    out << ans;
}

File: 2016 czerwiec/python/3.py
with open("liczby.txt", "r") as f:
    file = f.read().strip().split('\n')

ans = 0
with open("wyniki_6_3.txt", "w") as f:
    for s in file:
        if s[-1] == '2' and s[-2] == '0':
            ans += 1
    f.write(str(ans))